tests/resources/at-commands.mjs

```diff
@@ -1,259 +1,662 @@
+      jaws: [
+        keys.INSERT_TAB,
+        keys.INSERT_UP,
+        keys.INSERT_TAB,
+        keys.INSERT_UP,
+        keys.INSERT_TAB,
+        keys.INSERT_UP,
+        keys.INSERT_TAB,
+        keys.INSERT_UP,
+        keys.INSERT_TAB,
+        keys.INSERT_UP,
+        keys.INSERT_TAB,
+        keys.INSERT_UP,
+        keys.INSERT_TAB,
+        keys.INSERT_UP
+      ]
+    },
+    interaction: {
+      jaws: [
+        keys.INSERT_TAB,
+        keys.INSERT_UP,
+        keys.INSERT_TAB,
+        keys.INSERT_UP,
+        keys.INSERT_TAB,
+        keys.INSERT_UP,
+        keys.INSERT_TAB,
+        keys.INSERT_UP,
+        keys.INSERT_TAB,
+        keys.INSERT_UP,
+        keys.INSERT_TAB,
+        keys.INSERT_UP,
+        keys.INSERT_TAB,
+        keys.INSERT_UP
+      ]
+    }
+  },
+  "Reading menuitemradio": {
+    reading: {
+      jaws: [
+        keys.INSERT_TAB,
+        keys.INSERT_UP,
+        keys.INSERT_TAB,
+        keys.INSERT_UP,
+        keys.INSERT_TAB,
+        keys.INSERT_UP,
+        keys.INSERT_TAB,
+        keys.INSERT_UP,
+        keys.INSERT_TAB,
+        keys.INSERT_UP,
+        keys.INSERT_TAB,
+        keys.INSERT_UP,
+        keys.INSERT_TAB,
+        keys.INSERT_UP
+      ]
+    },
+    interaction: {
+      jaws: [
+        keys.INSERT_TAB,
+        keys.INSERT_UP,
+        keys.INSERT_TAB,
+        keys.INSERT_UP,
+        keys.INSERT_TAB,
+        keys.INSERT_UP,
+        keys.INSERT_TAB,
+        keys.INSERT_UP,
+        keys.INSERT_TAB,
+        keys.INSERT_UP,
+        keys.INSERT_TAB,
+        keys.INSERT_UP,
+        keys.INSERT_TAB,
+        keys.INSERT_UP
+      ]
+    }
+  },
+  "Reading menuitemcheckbox": {
+    reading: {
+      jaws: [
+        keys.INSERT_TAB,
+        keys.INSERT_UP,
+        keys.INSERT_TAB,
+        keys.INSERT_UP,
+        keys.INSERT_TAB,
+        keys.INSERT_UP,
+        keys.INSERT_TAB,
+        keys.INSERT_UP,
+        keys.INSERT_TAB,
+        keys.INSERT_UP,
+        keys.INSERT_TAB,
+        keys.INSERT_UP,
+        keys.INSERT_TAB,
+        keys.INSERT_UP
+      ]
+    },
+    interaction: {
+      jaws: [
+        keys.INSERT_TAB,
+        keys.INSERT_UP,
+        keys.INSERT_TAB,
+        keys.INSERT_UP,
+        keys.INSERT_TAB,
+        keys.INSERT_UP,
+        keys.INSERT_TAB,
+        keys.INSERT_UP,
+        keys.INSERT_TAB,
+        keys.INSERT_UP,
+        keys.INSERT_TAB,
+        keys.INSERT_UP,
+        keys.INSERT_TAB,
+        keys.INSERT_UP
+      ]
+    }
+  },
+  "Reading disabled menuitem": {
+    reading: {
+      jaws: [
+        keys.INSERT_TAB,
+        keys.INSERT_UP,
+        keys.INSERT_TAB,
+        keys.INSERT_UP,
+        keys.INSERT_TAB,
+        keys.INSERT_UP,
+        keys.INSERT_TAB,
+        keys.INSERT_UP,
+        keys.INSERT_TAB,
+        keys.INSERT_UP,
+        keys.INSERT_TAB,
+        keys.INSERT_UP,
+        keys.INSERT_TAB,
+        keys.INSERT_UP
+      ]
+    },
+    interaction: {
+      jaws: [
+        keys.INSERT_TAB,

```

Comment:
somehow this script is adding things multiple times! gotta remove these all and fix the script.

Reply:
oh, I missed this was fixed in the last commit!